src/components/user-table/UserTable.tsx

```diff
@@ -157,6 +190,17 @@ const UserTable = ({
   const toolbarVisibility =
     selected.length > 0 ? VisibilityEnum.Visible : VisibilityEnum.Hidden
 
+  const handleChangePage = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent> | null,
+    page: number
+  ) => {
+    console.log(page)
+  }
```

Comment:
Is this a test `console.log` or it's just mocked?

Reply:
yes, just mocked